### DISCUSSION TITLE
请问lantern客户端

### DISCUSSION THREAD

**Discussion by willyoung601** [getlantern/lantern-forum-cn#540] [General]
请问lantern客户端，包括win版本和Android版本会自动更新吗？还是需要定期手动更新？还有以上两个平台的最新版本号是多少呢?谢谢

**Comment by MaxMKMax** [comment]
https://github.com/getlantern/lantern

@willyoung601 您好. 这是下载最新版本链接. 一般会自动更新的, 但有时候还是手动检查比较好.

Window 7.8.6 Android 7.8.5

谢谢您对蓝灯感兴趣!

**Reply by willyoung601** [reply]
已经缴费2年，请问上面链接里只有7.8.5的源代码，没有*.exe文件提供下载安装吗？还是我没找对地方，请稍微详细点，谢谢
